content/sensu-go/6.5/release-notes.md

```diff
@@ -88,6 +89,42 @@ PATCH versions include backward-compatible bug fixes.
 Read the [upgrade guide][1] for information about upgrading to the latest version of Sensu Go.
 
 ---
+## 6.5.0 release notes
+
+**October 12, 2021** &mdash; The latest release of Sensu Go, version 6.5.0, is now available for download.
+
+Insert Release Synopsis.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.5.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][229]) Added support for [Sumo Logic metrics handlers][230] and [TCP stream handlers][231]. The [pipeline API][232] provides HTTP access for retrieving and configuring Sumo Logic metrics handlers and TCP stream handlers.
+- ([Commercial feature][229]) Added Prometheus metrics for TCP stream handlers:
+    - sensu_go_tcp_stream_handler_events: Total number of events handled by the TCP stream handler
+    - sensu_go_tcp_stream_handler_errors: The total number of errors produced by the TCP stream handler
+    - sensu_go_tcp_stream_handler_latency: Distribution of handler latencies, in milliseconds, for the TCP stream handler
+    - sensu_go_tcp_stream_handler_connection_acquisition_latency: Distribution of connection acquisition latencies (how long it takes to acquire a connection from the connection pool), in milliseconds, within the TCP stream handler
+- New [pipelines][233] resource allows you to specify event filters, mutators, and handlers in a single workflow instead of listing filters and mutators in handler definitions. You can reference pipelines in your check definitions. The [`/pipelines` API endpoint][234] provides HTTP access for retrieving pipeline data and configuring pipelines, and you can use [sensuctl][235] to manage pipelines.
+- Check definitions now include the [pipelines attribute][236] for specifying pipeline resources to use for the check's observability events.
+- [Event logging][237] is no longer a commercial-only feature.
+- You can now set sensuctl environment variables for a [single sensuctl command][243] or with [sensuctl configure][244].
+
+**IMPROVEMENTS:**
+
+- Added [platform metrics logging][238] to log core Sensu metrics in InfluxDB Line Protocol format, along with the `disable-platform-metrics`, `platform-metrics-log-file`, and `platform-metrics-logging-interval` backend configuration flags for managing the platform metrics logging feature.
+- Added environment variables `SENSU_BACKEND_ETCD_CLIENT_USERNAME` and `SENSU_BACKEND_ETCD_CLIENT_PASSWORD` for connecting to external etcd via username and password authentication instead of certificate authentication. There are no corresponding configuration flags &mdash; these configuration options must be set via environment variables.
```

Comment:
```suggestion
- Added environment variables `SENSU_BACKEND_ETCD_CLIENT_USERNAME` and `SENSU_BACKEND_ETCD_CLIENT_PASSWORD` for [connecting to external etcd via username and password authentication][245] instead of certificate authentication. There are no corresponding configuration flags &mdash; these configuration options must be set via environment variables.
```